srcpkgs/xearth/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'xearth'
+pkgname=xearth
+version=1.1
+revision=1
+build_style=gnu-make
+makedepends="libX11-devel libXt-devel"
+short_desc="Xearth sets the X root window to an image of the Earth"
+maintainer="IDF31 <idf31@protonmail.com>"
+license="custom"
```

Comment:
please provide a text of the license

Reply:
Do I provide the text in license variable?